.github/workflows/windows.yml

```
@@ -169,6 +169,15 @@ jobs:
           sed -i "s|%GITHUB_REPO%|$GITHUB_REPOSITORY|"       dest/README.txt
           # Create dir for zipping
           mv dest dosbox-staging-windows-${{ env.VERSION }}
+
+      - name: Windows Defender AV Scan
+        shell: powershell
+        run: |
+          $ErrorActionPreference = 'Stop'
+          $dosboxDir = "./dosbox-staging-windows-${{ env.VERSION }}"
+          & 'C:\Program Files\Windows Defender\MpCmdRun.exe' -Scan -ScanType 3 -File $dosboxDir
+          if( $LASTEXITCODE -ne 0 ) { Throw "Exit $LASTEXITCODE : Windows Defender found an issue" }
```

```suggestion
          if( $LASTEXITCODE -ne 0 ) { 
              Get-Content -Path $env:TEMP\MpCmdRun.log 
              Throw "Exit $LASTEXITCODE : Windows Defender found an issue" 
          }
```
This change will output the log file if it fails so we can get more details on the issue.

---

Committed; let's set what's going on. Thank you for the help again @dragonwolf83!